### ISSUE TITLE
Dynamic content function for color selection

### ISSUE THREAD

**Issue by pimpawell** [soflyy/oxygen-bugs-and-features#2356]
Dynamic content function for color selection.
A very important function that will speed up the workflow.

**Comment by antic1pate** [comment]
Yes I want to be able to set colors based on Metabox custom field

**Comment by shmaltz** [comment]
This is really needed and would be especially useful for elements in a repeater
